src/translations/fr/translation.json

```
@@ -1,300 +1,300 @@
-    "ID for remote control": "ID for remote control",
-    "Go to tarkov tools with another browser and enter this ID to control this page from there": "Go to tarkov tools with another browser and enter this ID to control this page from there",
-    "Flea Market": "Flea Market",
-    "Barter": "Barter",
-    "Icon": "Icon",
-    "Home": "Home",
-    "Discord Server": "Discord Server",
-    "Tarkov Tools API": "Tarkov Tools API",
-    "Nightbot integration": "Nightbot integration",
-    "StreamElements integration": "StreamElements integration",
-    "API Users": "API Users",
-    "API Latency in milliseconds": "API Latency in milliseconds",
-    "Current API Performance": "Current API Performance",
-    "Current Average Latency": "Current Average Latency",
-    "API Latency": "API Latency",
-    "Discord bot for your Discord": "Discord bot for your Discord",
-    "Game content and materials are trademarks and copyrights of Battlestate Games and its licensors. All rights reserved.": "Game content and materials are trademarks and copyrights of Battlestate Games and its licensors. All rights reserved.",
-    "Search item...": "Search item...",
-    "No hideout modules requires this item": "No hideout modules requires this item",
-    "Hideout Module": "Hideout Module",
-    "Level": "Level",
-    "Amount": "Amount",
-    "Support on Patreon": "Support on Patreon",
-    "Visualization of all ammo types in Escape from Tarkov, along with maps and other great tools": "Visualization of all ammo types in Escape from Tarkov, along with maps and other great tools",
-    "Visualization of all ammo types in Escape from Tarkov": "Visualization of all ammo types in Escape from Tarkov",
-    "Found In Raid": "Found In Raid",
-    "Yes": "Yes",
-    "No": "No",
-    "No quest requires this item": "No quest requires this item",
-    "Item": "Item",
-    "Tarkov server status": "Tarkov server status",
-    "This item can't be sold on the Flea Market": "This item can't be sold on the Flea Market",
-    "No flea price seen in the past 24 hours": "No flea price seen in the past 24 hours",
-    "Buyback ratio": "Buyback ratio",
-    "Sell to Trader": "Sell to Trader",
-    "Sell to Flea": "Sell to Flea",
-    "Buy on Flea": "Buy on Flea",
-    "Trader buy": "Trader buy",
-    "Grid slots": "Grid slots",
-    "Inner size": "Inner size",
-    "Slot ratio": "Slot ratio",
-    "Price per slot": "Price per slot",
+    "open this page in another browser or window and connect using this id": "ouvrez cette page dans un autre navigateur ou une autre fenêtre et connectez-vous en utilisant cet identifiant",
+    "ID for remote control": "Identifiant pour télécommande",
+    "Go to tarkov tools with another browser and enter this ID to control this page from there": "Accédez aux outils tarkov avec un autre navigateur et entrez cet identifiant pour contrôler cette page à partir de là",
+    "Flea Market": "Marché aux puces",
```

The game in French  uses single word for Flea Market

```suggestion
    "Flea Market": "Marché",
```